packages/cli/src/format-helpers.ts

```diff
@@ -112,7 +112,13 @@ export function standardFooter() {
         recursiveChildrenResults(result);
       }
     });
-    console.log(chalk.grey('────────────────────────────────────────────────────────────────────────────────'));
+    if(testRun.agents.length) {
+      console.log(chalk.grey('────────────────────────────────────────────────────────────────────────────────'));
+    }
+    if(testRun.status === 'failed' && testRun.error) {
+      errorLines(testRun.error).forEach((line) => console.log(chalk.red(line)));
```

Comment:
We should probably output an error here indicating that the test run couldn't be attempted because the bundle had an error in it. Otherwise, it just blops the error the console without context and that's pretty confusing:

![image](https://user-images.githubusercontent.com/4205/93806766-488b0200-fc0f-11ea-9059-04ebee231b5b.png)